docs/references/remotes.md

```diff
@@ -0,0 +1,283 @@
+
+
+>	- account_name: str
+>        The storage account name. This is used to authenticate requests
+>        signed with an account key and to construct the storage endpoint. It
+>        is required unless a connection string is given, or if a custom
+>        domain is used with anonymous authentication.
+>
+>    - account_key: str
+>        The storage account key. This is used for shared key authentication.
+>        If any of account key, sas token or client_id is specified, anonymous access
+>        will be used.
+>
+>    - sas_token: str
+>        A shared access signature token to use to authenticate requests
+>        instead of the account key. If account key and sas token are both
+>        specified, account key will be used to sign. If any of account key, sas token
+>        or client_id are specified, anonymous access will be used.
+>
+>    - request_session: Session
+>        The session object to use for http requests.
+>
+>    - connection_string: str
+>        If specified, this will override all other parameters besides
+>        request session. See
+>        http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/
+>        for the connection string format.
+>
+>    - credential: azure.core.credentials_async.AsyncTokenCredential or SAS token
+>        The credentials with which to authenticate.  Optional if the account URL already has a SAS token.
+>        Can include an instance of TokenCredential class from azure.identity.aio.
+>
+>    - blocksize: int
+>        The block size to use for download/upload operations. Defaults to hardcoded value of
+>        ``BlockBlobService.MAX_BLOCK_SIZE``
+>
+>    - client_id: str
+>        Client ID to use when authenticating using an AD Service Principal client/secret.
+>
+>    - client_secret: str
+>        Client secret to use when authenticating using an AD Service Principal client/secret.
+>
+>    - tenant_id: str
+>        Tenant ID to use when authenticating using an AD Service Principal client/secret.
+>
+>    - anon: boolean, optional
+>        The value to use for whether to attempt anonymous access if no other credential is
+>        passed. By default (``None``), the ``AZURE_STORAGE_ANON`` environment variable is
+>        checked. False values (``false``, ``0``, ``f``) will resolve to `False` and
+>        anonymous access will not be attempted. Otherwise the value for ``anon`` resolves
+>        to ``True``.
+>
+>    - default_fill_cache: bool = True
+>        Whether to use cache filling with open by default
+>
+>    - default_cache_type: string ('bytes')
+>        If given, the default cache_type value used for "open()".  Set to none if no caching
+>        is desired.  Docs in fsspec
+>
+>    - version_aware : bool (False)
+>        Whether to support blob versioning.  If enable this will require the
+>        user to have the necessary permissions for dealing with versioned blobs.
+>
```

Comment:
```suggestion
- `version_aware`: `bool` (default: `False`)

    Whether to support blob versioning. If enable this will require the user to have the
    necessary permissions for dealing with versioned blobs.

```